k8s/jobs/django-collectstatic-job.yml

```
@@ -0,0 +1,17 @@
+metadata:
+  name: django-collectstatic-job
+  namespace: default
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: django-collectstatic
+          image: registry.digitalocean.com/mindjunkies/django-mindjunkies-web:latest
+          command: [ "/bin/bash", "-c", "python manage.py collectstatic --noinput" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)